compiler/ast.nim

```diff
@@ -417,9 +417,12 @@ type
     tyVoid
       # now different from tyEmpty, hurray!
 
-static:
-  # remind us when TTypeKind stops to fit in a single 64-bit word
-  assert TTypeKind.high.ord <= 63
+    tyUncheckedArray
```

Comment:
Sorry, can you replace the existing ``tyUnused`` with ``tyUncheckedArray`` please?